### PR TITLE
[Feature] Support integration with mock mode

### PR DESCRIPTION
**Pull request by zhangmo8**
### Purpose

Integrated mock mode. In order to run the project without back-end service

image
image

### CODE REVIEW

File: paimon-web-ui-new/mock/mm.config.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    db: util.libObj.mockjs.mock({
+      'books': [
+        {
+          'id|+1': 1,
+          user: /\d\d/,
+          view: /\d\d\d\d/,
+          'type|1': [`js`, `css`, `html`],
+          'discount|1': [`0`, `1`],
+          author: {
+            'name|1': [`张三`, `李四`],
```
Comment (labbomb) [inline paimon-web-ui-new/mock/mm.config.js:59]: Delete this line.

**Review by labbomb — COMMENTED** [review]
PTAL.

**Review by labbomb — APPROVED** [review]
LGTM.

**Review by songjianet — APPROVED** [review]

**Review by SteNicholas — APPROVED** [review]
LGTM. Thanks for @zhangmo8 contribution of mock mode.

**Pull request merged**